app/util/JSONLD.scala

```diff
@@ -63,7 +63,8 @@ object JSONLD {
     )
 
     //convert metadata to json using implicit writes in Metadata model
-    val metadataJson = resourceJson ++ toJson(metadata).asInstanceOf[JsObject]
+    //include metadata ID in the json
+    val metadataJson = resourceJson ++ toJson(metadata).asInstanceOf[JsObject] ++ JsObject(Seq("id" -> JsString(metadata.id.toString())))
```

Comment:
```suggestion
    val metadataJson = JsObject(Seq("id" -> JsString(metadata.id.toString()))) ++ resourceJson ++ toJson(metadata).asInstanceOf[JsObject]
```

While it helped during review, it might be a bit confusing to see ID as the last field listed.

Reordering this statement slightly would place the ID first (after `@context`) instead  :+1:

Reply:
Altered... now `id` is right after `context`...actually im not sure how to make it the first...
```
{
        "@context": [
            "https://clowderframework.org/contexts/metadata.jsonld",
            {
                "Funding Institution": "http://sead-data.net/terms/FundingInstitution"
            }
        ],
        "id": "614e459819ec1c778a5ffd7b",
        "attached_to": {
            "resource_type": "cat:file",
            "url": "http://localhost:9000/files/6101852f19ec54b93cb07ff1"
        },
        "created_at": "Fri Sep 24 16:39:36 CDT 2021",
        "agent": {
            "@type": "cat:user",
            "name": "Chen Wang",
            "user_id": "http://localhost:9000/api/users/610183f740d74ef84576680d"
        },
        "content": {
            "Funding Institution": "ncsa"
        }
    },
```

Reply:
For JSON-LD documents it's good if the @context is always first.